client/src/services/tokens.js

```diff
@@ -2,16 +2,6 @@ import * as storage from '../utils/storage';
 
 import { ACCESS_TOKEN, REFRESH_TOKEN } from '../constants/storage';
 
-/**
- * Persist token to storage.
- *
- * @param {{accessToken, refresToken}} params
- */
-export function persist({ accessToken, refreshToken }) {
-  setAccessToken(accessToken);
-  setRefreshToken(refreshToken);
-}
-
 /**
  * Get access token from storage.
```

Comment:
Are you using any conventions for the function arrangement? Functions are moved from up to down

Reply:
Yeah the linter shown error for function usage before declaration